tests/test_optimizer.py

```diff
@@ -1537,3 +1537,39 @@ def gen_expr(depth: int) -> exp.Expression:
         self.assertEqual(4, normalization_distance(gen_expr(2), max_=100))
         self.assertEqual(18, normalization_distance(gen_expr(3), max_=100))
         self.assertEqual(110, normalization_distance(gen_expr(10), max_=100))
+
+    def test_unnest_subqueries_without_agg_fix(self):
```

Comment:
Can we add these tests to the `unnest_subqueries.sql` fixture? Is there a reason to put them here and do manual work?